core/src/main/java/contacts/core/Contacts.kt

```diff
@@ -147,14 +155,19 @@ interface Contacts {
 fun Contacts(
     context: Context,
     customDataRegistry: CustomDataRegistry = CustomDataRegistry(),
-    apiListenerRegistry: CrudApiListenerRegistry = CrudApiListenerRegistry()
-): Contacts = ContactsImpl(
-    context.applicationContext,
-    ContactsPermissions(context.applicationContext),
-    AccountsPermissions(context.applicationContext),
-    customDataRegistry,
-    apiListenerRegistry
-)
+    apiListenerRegistry: CrudApiListenerRegistry = CrudApiListenerRegistry(),
+    logger: Logger = EmptyLogger(),
+): Contacts {
+    val loggerRegistry = LoggerRegistry(logger)
```

Comment:
Great idea hiding the creation of the `LoggerRegistry`. It does not need to be exposed to consumers at this point. We might change this in the future if we decide to support multiple loggers and configurations that apply to all loggers. But this is perfect for its current state!

Reply:
I will actually also do the same for the `apiListenerRegistry: CrudApiListenerRegistry = CrudApiListenerRegistry()`. It does not need to be a parameter. It can be hidden.

Reply:
I made the update here https://github.com/vestrel00/contacts-android/commit/e321874e5c37739a6cb0a26ac479400a662b911e